### ISSUE TITLE
Issue adding more profiles

### ISSUE THREAD

**Issue by phn2211** [mammuth/ha-fritzbox-tools#136]
Hi, this is really a useful addon. I installed it last night several times, because I wonder myself how to add additional profiles in integration pane. I started with 2 profiles, jumped to 3 and 4 and now a fifth porfile is needed. Do i have to install again and type them into the field during installation or is it possible to add more profiles later?

**Comment by AaronDavidSchneider** [comment]
You will need to re-add the integration.

**Comment by phn2211** [comment]
Thanks for your quick reply.
Can u explain the process a little bit more detailed?

Do i have to uninstall the application in hacs or would itb be enough to
delete theb tools in integration panel?

:-)

Aaron David Schneider <notifications@github.com> schrieb am Mo., 28. Dez.
2020, 06:44:

**Comment by AaronDavidSchneider** [comment]
You only need to remove it from the integration pane and add it again thereafter. Usually not even necessary to restart home assistant.

**Comment by mammuth** [comment]
@phn2211 do you have any new insights / did you resolve the issue?

**Comment by oopsitsaflame** [comment]
I too think that a config dialog apart from the initial setup would be great. 
esp since config via configuration.yaml is the "old and outdated" method.

**Comment by Zahnweh** [comment]
@mammuth, adding a config option would be great. Not only for adding more profiles afterwards but also for adjusting which settings shall be controlled. If you decided not to control port forwardings during the initial installation and then change your mind you will always have to re-add the integration. This can be easier.

**Comment by AaronDavidSchneider** [comment]
@chemelli74 is currently working on merging fritzbox_tools into HomeAssistant.
This repository is likely not going to be maintained once the merge is complete.

**Comment by chemelli74** [comment]
Done, all merged. Now just wait for 2021.7.0 :-)

Simone

**Comment by chemelli74** [comment]
This custom integration is now deprecated because is fully merged into HomeAssistant core.

I would suggest you configure the new integration and retest your issue.
After reading the [documentation](https://www.home-assistant.io/integrations/fritz), if still present then open a issue on HomeAssistant Core repository.

Simone
